tests/docker/ducktape-deps.sh

```diff
@@ -161,4 +161,11 @@ function install_kafka_streams_examples() {
   mvn -DskipTests=true clean package
 }
 
+function install_arroyo() {
```

Comment:
I think we need to add a call to this in `vtools/qa/image/packer/install-deps.sh` for this dependency to be available in cdt.

Reply:
thanks, i have a branch with this change. but it looked like we need to merge this first because it calls ducktape-deps.sh, right?

Reply:
Yeah. This needs to be merged first. Was just pointing out that a change is needed in vtools too in case you weren't aware.

Reply:
thanks. i would have forgotten! here ya go https://github.com/redpanda-data/vtools/pull/1499